### PR TITLE
#8685cga5j GIS Content Updates

### PR DESCRIPTION
**Pull request by lc-hd**
This PR has a collection of content updates to improve the GIS UI.
To verify, you can confirm it meets the requirements of the Google Doc (not included)

### CODE REVIEW

**Comment by arojas1** [comment]
Task linked: [CU-8685cga5j GIS for community registration](https://app.clickup.com/t/8685cga5j)

**Review by arojas1 — APPROVED** [review]
Super minor change, NLD contact link should have the same link UI as everything else. Other than that LGTM!
image

File: localcontexts/static/css/register.css
Comment (birbjam) [inline localcontexts/static/css/register.css:1]: Screenshot 2024-02-13 at 1 40 01 PM

- This padding is making the buttons sit too close to the edge on the bottom.

- Native Land Digital link should have the class of `default-a` so it looks like all the other links.

**Review by birbjam — APPROVED** [review]
Two misc changes requested, otherwise it looks good.

**Pull request merged**